src/providers/providers/index.ts

```diff
@@ -146,12 +146,26 @@ export const BINANCECHAINWALLET: IProviderInfo = {
   check: "isBinanceChainWallet"
 };
 
+/**
+ * @deprecated Use CoinbaseWalletSdk
+ */
 export const WALLETLINK: IProviderInfo = {
   id: "walletlink",
   name: "Coinbase Wallet",
-  logo: WalletLinkLogo,
+  logo: CoinbaseWalletLogo,
   type: "qrcode",
-  check: "isWalletLink",
+  check: "isCoinbaseQr",
+  package: {
+    required: [["appName", "infuraId", "rpc"]]
+  }
+};
+
+export const COINBASEWALLET: IProviderInfo = {
+  id: "coinbasewallet",
+  name: "Coinbase Wallet",
+  logo: CoinbaseWalletLogo,
+  type: "qrcode",
+  check: "isCoinbaseQr",
```

Comment:
CBWSDK handles both injected and qrcode cases. we don't need to have a separate copy. “Connect to your Coinbase Wallet” should be fine.
```suggestion
  type: "injected", 
  check: "isCoinbaseWallet",
```